spec/system/school/customization_spec.rb

```diff
@@ -209,6 +225,59 @@ def image_path(filename)
 
   scenario 'user who is not logged in gets redirected to sign in page' do
     visit customize_school_path
-    expect(page).to have_text("Please sign in to continue.")
+    expect(page).to have_text('Please sign in to continue.')
+  end
+
+  scenario 'school admin rearrange and edit link title, url' do
```

Comment:
@developedBySJ The setup for this test can be made much cleaner and readable. In this example, you are creating the links using the UI and a number to identify the link. As we have already tested link creation, let's manually create those links and identify them by names, something like:

```ruby
      context 'there are already existing links in the school` do 
         let(:school_link_1) { create :school_link ..... }
         let(:school_link_2) { create :school_link ..... }
         
         scenario 'admin updates and changes order of links' do
             ## tests go here
         end
      
      end
```

User `Faker::Internet.url` to create the URL values for these school links.